core/lib/types/src/api/mod.rs

```diff
@@ -733,3 +735,67 @@ pub struct Proof {
     pub address: Address,
     pub storage_proof: Vec<StorageProof>,
 }
+
+/// Collection of overridden accounts, useful for `eth_estimateGas`.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct StateOverride(HashMap<Address, OverrideAccount>);
```

Comment:
could we move it to a new file? this file is already huge..

Reply:
Sure!